include/custom_op/custom_op_lite.h

```diff
@@ -11,6 +11,9 @@
 #include "ort_c_to_cpp.h"
 #include "onnxruntime_f16.h"
 
+#pragma warning(push)
+#pragma warning(disable : 4459)  // declaration of 'api' hides global declaration
```

Comment:
cannot disable the warning, which can be detected some security check tools, and it eventually become a server security issue in some dashboard.

Reply:
OK I'll see if there's other way to fix it